analyzers/src/SonarAnalyzer.SourceGenerators/SonarAnalyzer.SourceGenerators.csproj

```diff
@@ -20,4 +20,10 @@
       <TargetPathWithTargetPlatformMoniker Include="$(PKGNewtonsoft_Json)\lib\netstandard2.0\Newtonsoft.Json.dll" IncludeRuntimeDependency="false" />
     </ItemGroup>
   </Target>
+
+  <ItemGroup>
```

Comment:
This would be nicer after ItemGroup for PackageReference, as they are logically related